templates/man.mustache

```diff
@@ -8,6 +8,12 @@
 	{{shortdesc}}
```

Comment:
Instead of adding an entire section for this, what do you think about injecting a human-readable explanation of the hook into the `shortdesc`?

Reply:
@mrsdizzie Array seems fine to me.

Reply:
Made both of those changes and updated description. Hook details are now just part of the description